### PR TITLE
Update fleet-settings.asciidoc

### PR DESCRIPTION
**Pull request by yomduf**
The official documentation isn't clear on the fact that it's not possible to have a single fleet server for several elasticsearch clusters So adding an Elasticsearch outputs different from the cluster to which the fleet server belongs is not possible.

### CODE REVIEW

**Comment by jrodewig** [comment]
@kilfoyle Can you take a look at this when you have a moment? If the review will require 2+ hrs, please feel free to add it to the current docs sprint or add it to the backlog for next. Thanks!

**Review by kilfoyle — APPROVED** [review]
@yomduf Thanks for opening this! It looks like a really important tip.
Screenshot 2023-07-04 at 9 24 42 AM



I've rebased the PR onto the `main` branch and added a small change to the text. I'll merge and backport into the current docs.

One small tip for next time you open a docs PR: After you click the `edit` button, in the URL that opens, you can switch `8.8`, `current` or whatever branch to `main`, and that will make the PR much easier to merge.

For example, to update "[Migrate from Beats to Elastic Agent](https://www.elastic.co/guide/en/fleet/current/migrate-beats-to-agent.html)", the edit button automatically opens this URL:
`https://github.com/elastic/ingest-docs/edit/8.8/docs/en/ingest-management/migrate-beats-to-agent.asciidoc`

So you can change that to:
`https://github.com/elastic/ingest-docs/edit/main/docs/en/ingest-management/migrate-beats-to-agent.asciidoc`

And then the PR can be merged directly into the docs after one of us writers approves it.

Thanks for helping keep the docs in good shape!

**Comment by kilfoyle** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.9|[](https://github.com/elastic/ingest-docs/pull/305)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by kilfoyle** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.8|[](https://github.com/elastic/ingest-docs/pull/306)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**
